lib/pages/welcome_screen/components/welcome_foreground.dart

```diff
@@ -0,0 +1,106 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_movie_clean/components/primary_button.dart';
+import 'package:flutter_movie_clean/components/secondary_button.dart';
+import 'package:flutter_movie_clean/gen/assets.gen.dart';
+import 'package:flutter_movie_clean/gen/colors.gen.dart';
+import 'package:flutter_movie_clean/shared/extensions/context_ext.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:smooth_page_indicator/smooth_page_indicator.dart';
+
+class WelcomeForeground extends StatelessWidget {
+  const WelcomeForeground({
+    this.currentIndex = 0,
+    this.btnContinueClicked,
+    this.pageController,
+    Key? key,
+  }) : super(key: key);
```

Comment:
we can simply write like this
```suggestion
    super.key,
  })
```